runtime/crab/src/lib.rs

```diff
@@ -31,7 +31,7 @@ mod bridges_message;
 pub use bridges_message::*;
 
 mod migration;
-mod weights;
+pub mod weights;
```

Comment:
Any usage?

Reply:
Yes. It's used in the `block_execution_and_extrinsic_base_weight_should_match` test case.